kong/utils_test.go

```diff
@@ -12,8 +12,212 @@ import (
 	"github.com/google/go-cmp/cmp/cmpopts"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"github.com/tidwall/gjson"
 )
 
+var StatsDSchema = `{
```

Comment:
I believe this can be made `const`

```suggestion
const StatsDSchema = `{
```